source/includes/_custom_screeners.md

```diff
@@ -0,0 +1,197 @@
+# Custom Screeners
+
+A custom screener is targeting criteria that you can define yourself. These qualifications behave similarly to the qualifications available in the [/qualifications](http://docs.tapresearch.com/#list-all-qualifications) endpoint, however they are not re-usable. Each time a custom screener is used, it must be re-created.
+
+## Create a custom qualification
+
+Create a custom qualification question.
+
+> Sample Request 
+
+```json
+{
+    "name": "My qualification",
+    "question_text": "What do you want to do with your life?",
+    "answer_type": 0,
+    "country_language_id": 1
+}
+
+```
+
+> Sample Response 
+
+```json
+{
+    "name": "My qualification",
+    "question_id": 100112709,
+    "question_text": "What do you want to do with your life?",
+    "answer_type": 0,
+    "en_translation": "What do you want to do with your life?",
+    "country_language_id": 1
+}
+```
+
+### HTTP Request
+
+`POST https://www.tapresearch.com/api/v1/custom_qualifications`
+
+### Required Parameters
+Parameter | Type | Description
+--------- | ---- | -----------
+question_text | String | This is the question text that will be served to our respondents.
+answer_type | Integer | This value will determine how campaign_qualifications pre-codes are generated for quotas. See **Answer Types** section for more information.
+country_language_id | Integer | Country language targeting for the qualification.
+
+### Optional Parameters
+Parameter | Type | Description
+--------- | ---- | -----------
+name | String | Name of the custom qualification.
+country | String | Country that is being targeted.
+country_id | Integer | Country specific ID used for internal purposes.
+customer_id | Integer | The unique identifier associated with your account. See **Callbacks** section for more information.
```

Comment:
This is set automatically and users are unable to change this parameter

Reply:
Updated!